Monitor/src/models/Monitor.js

```diff
@@ -0,0 +1,45 @@
+const serviceClient = require('../clients/ServiceClient');
+const Discord = require('../clients/DiscordClient');
+
+
+class Monitor {
+  constructor() {
+    this.interval = 15000;
+    this.running = true;
+    this.livenessDetections = {};
+    this.timer = this.turnOn();
+  }
+
+   async livenessChecks(){
+    await this.livenessDetection(process.env.UNQfyBaseURL,'UNQfy');
+    await this.livenessDetection(process.env.NewsletterBaseURL,'Newsletter');
+  }
+
+  turnOn(){
+    setInterval(async ()=>{await this.livenessChecks()},this.interval);
```

Comment:
Creo que `timer` debería ser asignado aca. Si se prende y se apaga, `this.timer` no es mas el intervalo y nunca se va a poder volver a apagar.

Reply:
Resuelto en 994bfd2